bootstrap/bootstrap_peers.cpp

```diff
@@ -103,6 +105,18 @@ bootstrap_peers::initialize_peer_list(const Json::Value& root, bzn::peers_list_t
             continue;
         }
 
+        if (this->is_security_enabled())
+        {
+            // At this point we cannot validate uuids as we do not have
+            // signatures for all of them, so we simply ignore blacklisted
+            // uuids
+            if(bzn::utils::blacklist::is_blacklisted(uuid))
```

Comment:
space after if

Reply:
Right! OK, as penance I have scoured the code base and corrected this everywhere I found it.